Name the timer wakeup threads to distinguish them from their parent t…

jhelovuo opened the pull request:
…hread. This is useful in debugging.

jhelovuo committed:
Name the timer wakeup threads to distinguish them from their parent thread. This is useful in debugging.

dimbleby commented:
Seems fine, but a couple of asks please (sorry!)

- I think I prefer `format` as a more idiomatic way to construct this name, as in
```
.name(format!("mio_extras::timer : {}", thread::current().name().unwrap_or("no name")))
```

- Since I last did anything with this repository, travis CI has withdrawn its free service.  I've now updated master so that it uses github CI instead, can you please rebase on master and fix up?  This should just mean running `cargo fmt`, I don't expect it to show any other issues.

Do you care about getting this released?  I wouldn't want to give anyone the impression that this repository is active all of a sudden, but it has been two years since the last tag so I guess publishing would not be unreasonable.

jhelovuo committed:
More idiomatic text formatting for thread naming.

jhelovuo commented:
Ok, fixed text formatting and rebased to master.

It is not important for me to get this published soon. I just made this improvement to help my own debugging, but I got that case resolved already.

dimbleby commented:
thanks!
